src/cart/line-item.ts

```diff
@@ -38,6 +38,7 @@ export interface LineItem {
     name: string;
     url: string;
     quantity: number;
+    brand: string;
```

Comment:
Why does it needs to be behind an `include`? AFAIK the only reason we use includes is to avoid slow services or huge response sizes. `brand` seems small enough 😕

Reply:
Yeah, I changed my thinking on that earlier and just forgot about this comment. 😄